tests/cypress/support/commands.js

```diff
@@ -359,3 +359,103 @@ Cypress.Commands.add('logoutFromPublisher', () => {
+    cy.visit(`${Utils.getAppOrigin()}/publisher/apis`);
+
+    //select rest-api option from the menu item
+    cy.get('#itest-rest-api-create-menu', { timeout: 30000 });
+    cy.get('#itest-rest-api-create-menu').click();;
+    cy.get('#itest-id-landing-rest-create-default').click();
+    cy.get('#itest-id-apiname-input').type('ThirdPartyApi');
+    cy.get('#itest-id-apicontext-input').type('/thirdpartyapi');
+    cy.get('#itest-id-apiversion-input').type('1.0.0');
+    cy.get('#itest-id-apiendpoint-input').type(`${Utils.getAppOrigin()}/am/sample/thirdpartyapi/v1/api`);
+    cy.get('#itest-create-default-api-button').click();
+
+    //Mark as third party api
+    cy.get('#itest-api-details-portal-config-acc').click();
+    cy.get('#left-menu-itemDesignConfigurations').click();
+    cy.get('[name="advertised"]:first').click();
+    cy.get('[name="apiExternalProductionEndpoint"]').type(`${Utils.getAppOrigin()}/am/sample/thirdpartyapi/v1/externalapi`);
+    cy.get('[name="apiExternalSandboxEndpoint"]').type(`${Utils.getAppOrigin()}/am/sample/thirdpartyapi/v1/externalapi`);
+    cy.get('[name="originalDevPortalUrl"]').type('http://www.mocky.io/v2/5ec501532f00009700dc2dc1');
+    cy.get('#design-config-save-btn').click();
+    cy.get('#itest-api-details-portal-config-acc').click();
+
+    //publish
+    cy.get('#left-menu-itemlifecycle').click();
+    cy.get('[data-testid="Publish-btn"]').should('exist');
+    cy.get('[data-testid="Deploy as a Prototype-btn"]').should('exist');
+    cy.get('[data-testid="Publish-btn"]').click();
+
+    //check if the api is third-party and published
+    cy.get('[data-testid="itest-api-state"]').contains('PUBLISHED').should('exist');
+    cy.get('[data-testid="itest-third-party-api-label"]').contains('Third Party').should('exist');
+
+    //Check if the subscriptions,runtime config, resources, scopes, monetization, test console sections are present
+    cy.get('#itest-api-details-portal-config-acc').click();
+    cy.get('#left-menu-itemsubscriptions').should('exist');
+    cy.get('#left-menu-itemsubscriptions').click();
+    cy.get('[name="Unlimited"]').click();
+    cy.get('#subscriptions-save-btn').click();
+    cy.get('#itest-api-details-portal-config-acc').click();
+    cy.get('#itest-api-details-api-config-acc').click();
+    cy.get('#left-menu-itemRuntimeConfigurations').should('exist');
+    cy.get('#left-menu-itemresources').should('exist');
+    cy.get('#left-menu-itemLocalScopes').should('exist');
+    cy.get('#left-menu-itemMonetization').should('exist');
+    cy.get('#itest-api-details-api-config-acc').click();
+    cy.get('#left-menu-itemTestConsole').should('exist');
+
+    //Check if the api is not deployable
+    cy.get('#left-menu-itemdeployments').click();
+    cy.get('[data-testid="third-party-api-deployment-dialog"]').contains('This API is marked as a third party API. The requests are not proxied through the gateway. Hence, deployments are not required.').should('exist');
```

Comment:
Line length exceeds 120 chars. Please check other places as well.

Reply:
This is fixed in [apim-apps#426](https://github.com/wso2/apim-apps/pull/426)